### PR TITLE
Update instructions for MacOS

### PR DESCRIPTION
**Pull request by alexmitchelldev**
Added instructions to help MacOS users that may receive errors when trying to build the tests.

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -99,3 +99,7 @@ $ cmake -G Xcode ..
 This example generates the XCode files so that you can open the project in XCode.
 With the project opened in XCode you may need to add another target. 
 For a new target go to File -> New -> Target from the menu, _OS X \ Application \ Command Line Tool_ -> Next _C++_ as the language and provide a target name. Run the code using the target.
+
+If your terminal throws an `Xcode x.x not supported` error, it is highly likely that Xcode is not installed on your machine or that  your version is out of date.
```
Comment (vaeng) [inline docs/TESTS.md:103]:
```suggestion
If your terminal throws an `Xcode x.x not supported` error, it is highly likely that Xcode is not installed on your machine, or that your version is out of date.
```

File: docs/TESTS.md [hunk 2]
```diff
@@ -99,3 +99,7 @@ $ cmake -G Xcode ..
...
+Please download and install Xcode via the App Store or via [this link](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
```
Comment (vaeng) [inline docs/TESTS.md:104]: Can you please change your links to reference links?

https://exercism.org/docs/building/markdown/markdown

**Review by vaeng — COMMENTED** [review]
Thanks for the changes. I am sure Mac users will be glad to not run into the same errors you navigated.

File: docs/TESTS.md (hunk 2 above)
Comment (alexmitchelldev) [reply docs/TESTS.md]: Sure, done :)

**Review by vaeng — APPROVED** [review]
Thanks for the quick work!

**Pull request merged**